scripts/cut_tiles.sh

```diff
@@ -147,13 +184,9 @@ if  [ -n "$S3_PATH" ]; then
     #clean up s3 old files
     clean_s3 ${S3_PATH} 30
     #push up s3 new files
-    for f in ${cur_extras_dir}/*; do
-      aws s3 mv ${f} ${S3_PATH} --acl public-read
-    done
-    #signal other stacks to get new data
-    aws s3 ls ${S3_PATH}/planet_${stamp}.tar
+    aws s3 cp --recursive ${CUR_PLANET_DIR} ${S3_PATH} --acl public-read
```

Comment:
There shouldn't be any need to set the acl in the cp command, I've already made the buckets public. Otherwise looks fine.